lib/linter.js

```diff
@@ -75,6 +75,7 @@ exports.resultSeverity = {
 exports.lint = function (input, fullPath, config) {
     var results = [];
     var ast;
+    var extend = require('lodash.assign');
```

Comment:
I really think we should just use `lodash.merge()` instead. It seems really unnecessary to include a second (almost identical) dependency. Could we also keep the `require()` calls grouped at the top?